### ISSUE TITLE
xpra: lacks h264

### ISSUE THREAD

**Issue by Gottox** [voidlinux/void-packages#4314]
The makedepends list x264-devel, but the resulting package does not include h264 support.

```
~ xpra attach :7 --encoding=h264

** (process:9303): WARNING **: Trying to register gtype 'GMountMountFlags' as enum when in fact it is of type 'GFlags'

** (process:9303): WARNING **: Trying to register gtype 'GDriveStartFlags' as enum when in fact it is of type 'GFlags'

** (process:9303): WARNING **: Trying to register gtype 'GSocketMsgFlags' as enum when in fact it is of type 'GFlags'
Gtk-Message: Failed to load module "canberra-gtk-module"
2016-06-10 11:08:52,828 Xpra gtk2 client version 0.17.3-r12738
2016-06-10 11:08:52,829  running on Linux
2016-06-10 11:08:52,830 Warning: failed to import opencv:
2016-06-10 11:08:52,830  No module named cv2
2016-06-10 11:08:52,831  webcam forwarding is disabled
2016-06-10 11:08:53,131 GStreamer version 1.8 for Python 2.7
2016-06-10 11:08:53,153 Warning: csc_cython failed its self test
2016-06-10 11:08:53,153  No module named numpy
invalid encoding: h264
gtk2 xpra client supports the following encodings:
 * vp8         VP8 video codec
 * vp9         VP9 video codec
 * png         Portable Network Graphics (lossless, 24bpp or 32bpp for transparency)
 * png/P       Portable Network Graphics (lossy, 8bpp colour)
 * png/L       Portable Network Graphics (lossy, 8bpp grayscale)
 * webp        WebP compression (lossless or lossy)
 * rgb         Raw RGB pixels, lossless, compressed using zlib or lz4 (24bpp or 32bpp for transparency)
 * jpeg        JPEG lossy compression
```

**Comment by Gottox** [comment]
EOFFBYONE #4313

**Comment by chneukirchen** [comment]
Hm, it links against libx264.so.142 tho.

**Comment by chneukirchen** [comment]
Fixed in 81bb3dc801e
